Makefile

```diff
@@ -192,3 +192,8 @@ update-all-go-deps:
 		$(GO) get -d $$m; \
 	done
 	@cd ./documentation/examples/remote_storage/ && $(GO) mod tidy
+
+.PHONY: check-runtime-versions
```

Comment:
I suggest we remove the `exit 1` (turn them into warnings) and run the target as part of `build`, otherwise it can easily be overlooked.
("warnings" in case our min version is too strict or in case the script produces false positives)

Reply:
Removed the exit 1.

Reply:
I wouldn't consider adding a warning line to the current build logs, if there is a versions mismatch, excessive. It'd be helpful.

Having it as an independent target means it'll never be used when really needed (during the build).